checks/raw/signed_releases.go

```diff
@@ -15,15 +15,20 @@
 package raw
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/ossf/scorecard/v5/checker"
+	"github.com/ossf/scorecard/v5/clients"
 )
 
 // SignedReleases checks for presence of signed release check.
 func SignedReleases(c *checker.CheckRequest) (checker.SignedReleasesData, error) {
 	releases, err := c.RepoClient.ListReleases()
 	if err != nil {
+		if errors.Is(err, clients.ErrUnsupportedFeature) {
+			return checker.SignedReleasesData{}, nil
+		}
```

Comment:
I'm not sure we should change this one. the Signed-Releases check relies on this data alone.